Rego/OneDriveConfig.rego

```diff
@@ -90,50 +124,54 @@ tests[{
 # Baseline 2.3: Policy 1
 #--
 ReportDetails2_3(Policy) = Description if {
-    Policy.DefaultSharingLinkType != 3
+    Policy.OneDriveLoopSharingCapability != 2
+    Description := "Requirement met: Anyone links are disabled"
+}
+
+ReportDetails2_3(Policy) = Description if {
+    Policy.OneDriveLoopSharingCapability == 2
     Policy.FileAnonymousLinkType == 1
     Policy.FolderAnonymousLinkType == 1
 	Description := "Requirement met"
 }
 
 ReportDetails2_3(Policy) = Description if {
-    Policy.DefaultSharingLinkType != 3
+    Policy.OneDriveLoopSharingCapability == 2
     Policy.FileAnonymousLinkType == 2
     Policy.FolderAnonymousLinkType == 2
 	Description := "Requirement not met: both files and folders are not limited to view for Anyone"
 }
 
 ReportDetails2_3(Policy) = Description if {
-    Policy.DefaultSharingLinkType != 3
+    Policy.OneDriveLoopSharingCapability == 2
     Policy.FileAnonymousLinkType == 1
     Policy.FolderAnonymousLinkType == 2
 	Description := "Requirement not met: folders are not limited to view for Anyone"
 }
 
 ReportDetails2_3(Policy) = Description if {
-    Policy.DefaultSharingLinkType != 3
+    Policy.OneDriveLoopSharingCapability == 2
     Policy.FileAnonymousLinkType == 2
     Policy.FolderAnonymousLinkType == 1
 	Description := "Requirement not met: files are not limited to view for Anyone"
 }
 
-ReportDetails2_3(Policy) = Description if {
-    Policy.DefaultSharingLinkType == 3
-	Description := "Requirement not met: default link sharing type is set to Anyone with link"
-}
-
 tests[{
     "Requirement" : "Anyone link permissions SHOULD be limited to View",
     "Control" : "OneDrive 2.3",
     "Criticality" : "Should",
     "Commandlet" : ["Get-SPOTenant"],
-    "ActualValue" : [Policy.DefaultSharingLinkType, Policy.FileAnonymousLinkType, Policy.FolderAnonymousLinkType],
+    "ActualValue" : [Policy.OneDriveLoopSharingCapability, Policy.FileAnonymousLinkType, Policy.FolderAnonymousLinkType],
     "ReportDetails" : ReportDetails2_3(Policy),
     "RequirementMet" : Status
 }] {
     Policy := input.SPO_tenant_info[_]
-	Conditions := [Policy.DefaultSharingLinkType != 3, Policy.FileAnonymousLinkType == 1, Policy.FolderAnonymousLinkType == 1]
-    Status := count([Condition | Condition = Conditions[_]; Condition == false]) == 0
+        Conditions1 := [Policy.OneDriveLoopSharingCapability !=2]
```

Comment:
There is extra spacing on this line to line 173

Reply:
thank you for pointing it out, changed